src/Facility.Definition/CodeGen/CodeGenUtility.cs

```diff
@@ -15,12 +16,20 @@ public static class CodeGenUtility
 	/// <summary>
 	/// Capitalizes the specified string.
 	/// </summary>
+#if NET6_0_OR_GREATER
+	public static string Capitalize(string value) => value.Length == 0 ? value : string.Concat(value[..1].ToUpperInvariant(), value.AsSpan(1));
+#else
 	public static string Capitalize(string value) => value.Length == 0 ? value : value.Substring(0, 1).ToUpperInvariant() + value.Substring(1);
+#endif
```

Comment:
This code complexity seems a little unnecessary for a code-generation tool; the wins from making a `Span` instead of a `Substring` seem like they would be minimal.

Reply:
True! But it's either that or disable the analyzer warning.

Reply:
Ah.